owl/SOMA-HOME.owl

```diff
@@ -0,0 +1,553 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/soma-home.owl#"
```

Comment:
better use uppercase latter, not sure if it matters or not.
so use *SOMA-HOME.owl* exactly as I wrote it here.